cmd/release-controller/main.go

```diff
@@ -236,6 +234,9 @@ func (o *options) Run() error {
 		o.ArtifactsHost,
 		releaseInfo,
 		graph,
+		pluginAgent,
```

Comment:
You'll want to look at how the auditor is created below but you'll just be an optional component that can run in certain modes.  Unlike audit you can be run in the main sync path, so you'll have an if block down below where you start your loop (i.e. something like RunMonitorAccepted)